lib/YaST/Bootloader/BootloaderController.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Controller for YaST bootloader module.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::Bootloader::BootloaderController;
+use strict;
+use warnings;
+use YuiRestClient;
+use YaST::Bootloader::BootcodeOptionsPage;
+use YaST::Bootloader::BootloaderPage;
+use testapi;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {}, $class;
+    return $self->init($args);
+}
+
+sub init {
+    my ($self, $args) = @_;
+    $self->{BootcodePage}  = YaST::Bootloader::BootcodeOptionsPage->new({app => YuiRestClient::get_app()});
+    $self->{BooloaderPage} = YaST::Bootloader::BootloaderPage->new({app => YuiRestClient::get_app()});
+    return $self;
+}
+
+sub get_bootcode_options_page {
+    my ($self) = @_;
+    $self->{BootcodePage};
+}
+
+sub get_bootloader_page {
+    my ($self) = @_;
+    $self->{BootloaderPage};
+}
+
+sub get_ui_defaults {
```

Comment:
As it shows currently set values, I would rename it to reflect that. It shows defaults only till the moment we change them. So I would propose e.g. `get_current_settings`